src/emissions-workspace/emissions-checks.service.ts

```diff
@@ -16,17 +17,67 @@ export class EmissionsChecksService {
     }
   }
 
-  runChecks(payload: EmissionsImportDTO)  {
+  runChecks(payload: EmissionsImportDTO): string[] {
     this.logger.info('Running Emissions Import Checks');
 
     const errorList: string[] = [];
-    const weeklyTestSummaryCheckErrors = this.weeklyTestSummaryCheckService.runChecks(payload);
+    const weeklyTestSummaryCheckErrors = this.weeklyTestSummaryCheckService.runChecks(
+      payload,
+    );
+    const invalidDatesCheckErrors = this.invalidDatesCheck(payload);
 
-    errorList.push(...weeklyTestSummaryCheckErrors);
+    errorList.push(...weeklyTestSummaryCheckErrors, ...invalidDatesCheckErrors);
 
     this.throwIfErrors(errorList);
     this.logger.info('Completed Emissions Import Checks');
 
     return errorList;
   }
+
+  // IMPORT-23
+  invalidDatesCheck(payload: EmissionsImportDTO): string[] {
+    let earliestDate: number = undefined;
```

Comment:
Nothing wrong with this though personally I always feel like its extra code when we make a new variable = undefined since that would be the default if we don't set it as anything.